.eslintrc.json

```diff
@@ -1,34 +1,33 @@
 {
-    "settings": {
-        "react": {
-          "version": "detect"
-        }
-    },
-    "env": {
-        "browser": true,
-        "node": true,
-        "es2021": true
-    },
-    "extends": ["eslint:recommended", "plugin:react/recommended", "plugin:prettier/recommended"],
-    "parserOptions": {
-        "ecmaFeatures": {
-            "jsx": true
-        },
-        "ecmaVersion": 12,
-        "sourceType": "module"
-    },
-    "plugins": [
-        "react"
-    ],
-    "rules": {
-        "semi": [2, "always"],
-        "react/react-in-jsx-scope": "off",
-        "prettier/prettier": [
-            "error",
-            {
-              "endOfLine": "auto"
-            }
-          ],
-        "react/prop-types": 0
-    }
+	"settings": {
```

Comment:
I don't know what you changed here, but discard those changes

Reply:
Done